lib/board.rb

```diff
@@ -1,13 +1,44 @@
+# frozen_string_literal: true
 # Use this as a sketch of how your Board class could look, you can fill in these
 # methods, or make a different class from scratch
+require 'tty-table'
+
 class Board
+  attr_reader :table
+  attr_accessor :rows
+
   def initialize
+    generate_board
   end
 
   def drop_checker(color, column)
   end
 
+  def generate_board
+    rows = []
+    6.times do
+      rows << []
+    end
+
+    rows.each do |r|
+      7.times do
+        r << empty_string
+      end
+    end
+    @rows = rows
+  end
+
+  def empty_string
+    "     ".freeze
```

Comment:
What do you think of having a true empty string instead of 5 spaces? In my version, I filled my grid with nil. Later when someone is trying to test whether a position is empty or not they could use .empty? for a true empty string or .nil? for nil. Using .empty? on '     ' evaluates to false, the same as if something like 'red' or 'black' was in that position.

Reply:
I'm not really committed to this part of the implementation, so I'm definitely up for updating it.